src/game-flow/game-flow.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@nestjs/common';
+import { CreateGameFlowDto } from './dto/create-game-flow.dto';
+import { UpdateGameFlowDto } from './dto/update-game-flow.dto';
+import { Game } from 'holdem-poker';
+
+@Injectable()
+export class GameFlowService {
+  game: any = new Game([100, 100, 100, 100], 10);
+  clientToUser = {};
+
+  create(createGameFlowDto: CreateGameFlowDto, clientId: string) {
+    this.game.startRound();
+    console.log('clientId ', clientId);
```

Comment:
I know how logging could be helpful for node.js but there are better tools. Checkout this
https://github.com/winstonjs/winston